### PR TITLE
Announcement for 2.3.12

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

File: news/_posts/2015-07-09-akka-2.3.12-released.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+*Dear hAkkers,*
+
+We—the Akka committers—are pleased to be able to announce the availability of Akka 2.3.12. This is the twelfth maintenance release of the 2.3 branch. This release contains a few important fixes:
+
+ - a fix for a race between Terminated message and final write in TCP ([#17932](https://github.com/akka/akka/issues/17932))
+ - a fix for CircuitBreaker where concurrent transitions did not occur ([#17415](https://github.com/akka/akka/issues/17415))
+ - improvement in flow control of system message delivery ([#17554](https://github.com/akka/akka/issues/17554))
+
+This release also [adds a configurable TestActor creation timeout in TestKit](https://github.com/akka/akka/issues/17711), [suppresses Tcp.CloseCommand from logging](https://github.com/akka/akka/issues/17572) and improves documentation ([#17669](https://github.com/akka/akka/issues/17669) [#15569](https://github.com/akka/akka/issues/15569)).
```
Comment (ktoso) [inline news/_posts/2015-07-09-akka-2.3.12-released.md]: add coma: `([#17669](https://github.com/akka/akka/issues/17669), [#15569](https://github.com/akka/akka/issues/15569)).`

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM

**Comment by 2m** [comment]
Thanks gyus!

**Pull request merged**